### PR TITLE
Add matches to abac ology 673

### PR DESCRIPTION
**Pull request by sgrogg72**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -178,6 +178,13 @@ const compare = (condition, value, attributes) => {
       if (compareValue === undefined) return undefined;
       return Array.isArray(value) && Array.isArray(compareValue) && value.every(x => compareValue.includes(x));
 
+    case 'matches':
+      if (compareValue === undefined) return undefined;
+      // we want to construct express here
```
Comment (jagoda) [inline src/index.js:183]: Typo?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/abac",
-  "version": "4.3.1",
+  "version": "4.4.0",
```
Comment (jagoda) [inline package.json:3]: It looks like you will need to push a tag to really publish. Might be nice to bump the version and push the tag in the same push to keep things consistent.

Comment (jagoda) [reply package.json:3]: I think our OSS packages are still doing this directly in `master` after the PR merge.

**Review by macnaughton — APPROVED** [review]
LGTM, open to discussion on ReDoS mitigation

**Review by bishopb — APPROVED** [review]
LGTM

**Review by joedimarzio — APPROVED** [review]

**Review by jagoda — APPROVED** [review]

**Review by mdlavin — APPROVED** [review]

**Pull request merged**